src/app/multi-editor/multi-editor.component.ts

```diff
@@ -61,12 +60,11 @@ export class MultiEditorComponent implements OnInit {
   }
 
   onSave() {
-    this.queryService.save(this.previewedActions, this.checkedRecords)
-    .then((res) => {
-      this.successMessage = res['message'];
-      this.totalRecords = -1;
-      this.changeDetectorRef.markForCheck();
-    })
+    let checkedUUIDs: string[];
+    this.allSelected ? checkedUUIDs = [] :
```

Comment:
remove this line

Reply:
If everything is selected then I dont need to send to the backend any UUIDS.